src/controllers/googleFacebook.js

```diff
@@ -20,7 +20,10 @@ export const userToken = async (req, res) => {
     lastname: req.user.lastName,
     email: req.user.email,
   };
+
   const token = await storeToken(user);
+  if (req.user.socialAuth) return res.redirect(`http://localhost:8080?accessToken=${token.accessToken}&refreshToken=${token.refreshToken}`);
+
```

Comment:
the url should be an environment variable.
It might change again and again.
It won't be using localhost in production as well